IFC4x3/Properties/t/ThicknessEnd_23IeYdlhfElwvDUY45W8gF/Documentation.md

```diff
@@ -0,0 +1 @@
+The nominal thickness of the pavement after a transition from its original value given as Thickness. The size information is provided in addition to the shape representation and the geometric parameters used within. In case of inconsistency between the geometric parameters and the size properties, provided in the attached property set, the geometric parameters take precedence. ThicknessEnd location shall be regarded as the pavement object’s placement plus its Length.
```

Comment:
'plus its Length' which Length is meant here?`Nominal or from geometric representation? Also, can it be clearly extracted from Representation? Just being careful for the sake of unambiguity.